lib/restate-constructs/register-service-handler/entrypoint.mts

```diff
@@ -0,0 +1,199 @@
+// from https://github.com/aws/aws-cdk/blob/main/packages/%40aws-cdk/custom-resource-handlers/lib/core/nodejs-entrypoint-handler/index.ts
```

Comment:
Do we need to this anywhere in our license notices, as [CDK is APL2](https://github.com/aws/aws-cdk/?tab=Apache-2.0-1-ov-file#readme)?